src/d3a/d3a_core/export.py

```diff
@@ -132,6 +133,22 @@ def export(self, export_plots=True, power_flow=None):
     def data_to_csv(self, area, is_first):
         self._export_area_with_children(area, self.directory, is_first)
 
+    def area_tree_summary_to_json(self, data: Dict):
+        subdirectory = pathlib.Path(self.directory, "raw_data")
+        if not subdirectory.exists():
+            subdirectory.mkdir(exist_ok=True, parents=True)
+        json_file = os.path.join(self.directory, "area_tree_summary.json")
+        with open(json_file, 'w') as outfile:
+            json.dump(data, outfile, indent=2)
```

Comment:
I am still not sure whether JSON is the right format for buffering the results here. I thought about pickle but this will lead to issues at some point (from my experience and apparently also others: https://www.benfrederickson.com/dont-pickle-your-data/#:~:text=Just%20use%20JSON&text=Pickle%20on%20the%20other%20hand,data%20they%20can%20write%20out. ) 
Maybe @spyrostz  will have a better idea or will just say: JSON is good! Lets wait for that.

Reply:
Will wait for Spyros feedback on that!

Reply:
Better to avoid pickle altogether here actually. In addition to the points that the blog that Hannes added correctly mentions, I have also encountered another issue, which is that it does not work across different Python versions (Python 2 pickle cannot be read by Python 3 pickle, and this behavior exists even across 3.6 and 3.8 versions). 

TLDR: JSON is great for this purpose